Formula/packer.rb

```diff
@@ -16,34 +14,16 @@ class Packer < Formula
   end
 
   depends_on "go" => :build
+  depends_on "gox" => :build
   depends_on "govendor" => :build
 
-  go_resource "github.com/mitchellh/gox" do
-    url "https://github.com/mitchellh/gox.git",
-        :revision => "c9740af9c6574448fd48eb30a71f964014c7a837"
-  end
-
-  go_resource "github.com/mitchellh/iochan" do
-    url "https://github.com/mitchellh/iochan.git",
-        :revision => "87b45ffd0e9581375c491fef3d32130bb15c5bd7"
-  end
-
   def install
     ENV["XC_OS"] = "darwin"
     ENV["XC_ARCH"] = MacOS.prefer_64_bit? ? "amd64" : "386"
     ENV["GOPATH"] = buildpath
-    # For the gox buildtool used by packer, which
-    # doesn't need to be installed permanently.
-    ENV.append_path "PATH", buildpath
 
     packerpath = buildpath/"src/github.com/hashicorp/packer"
     packerpath.install Dir["{*,.git}"]
-    Language::Go.stage_deps resources, buildpath/"src"
-
-    cd "src/github.com/mitchellh/gox" do
-      system "go", "build"
-      buildpath.install "gox"
-    end
 
     cd packerpath do
       # We handle this step above. Don't repeat it.
```

Comment:
This comment seems like it needs to be revised

Reply:
`These are provided by the gox formula and upstream vendoring`?

Reply:
I don't know why we don't need stringer vendored. I guess the build never actually uses it?

Maybe just
```
# Avoid running `go get`
```
and leave the rest to our imagination